pages/[slug]/itinerary.js

```diff
@@ -121,13 +142,20 @@ export default function Itinerary({ trips, sights, setSights }) {
           ))}
         </StyledItinerary>
       </Scrollbox>
-      <StyledFooter>
-        <StyledLink href="/">Home</StyledLink>
-      </StyledFooter>
+      <Footer buttonlink={`/${currentTrip.slug}`} buttontext="Overview" />
     </>
   );
 }
 
+const StyledLottie = styled(Lottie)`
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 50vw;
+  height: 50vw;
+`;
+
 const StyledHeading1 = styled.h1`
```

Comment:
A better name for this styled component would be `Headline`. :)

Reply:
I checked on leo.org before, because I wasn't sure which one to use and it said that headline is more like "Schlagzeile". Not sure 🤷